<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmess</class>
 <widget class="QMainWindow" name="Qmess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="labName">
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtName">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>38</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ipLayout">
     <item>
      <widget class="QLabel" name="labIP">
       <property name="text">
        <string>Your Address:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labIPAdress">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>92</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="btnLogin">
       <property name="text">
        <string>Login</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnLogout">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextBrowser" name="browserMessage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>471</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="listOnlineUser">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>201</width>
      <height>171</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>OnlineUsers:</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="ipFinalLayout">
     <item>
      <widget class="QLabel" name="labFinalIP">
       <property name="text">
        <string>IP args</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFileIP"/>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFilePort"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>380</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="fileLayout">
     <item>
      <widget class="QPushButton" name="btnSendFile">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStop">
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="ProgressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>201</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="edtMessage">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>420</y>
      <width>471</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacerSend">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSendMessage">
       <property name="text">
        <string>Send Message</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>201</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labMask">
       <property name="text">
        <string>Mask</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="boxMask">
       <item>
        <property name="text">
         <string>255.255.255.255</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>192.168.255.255</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="btnListen">
       <property name="text">
        <string>Listen</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnChooseFile">
       <property name="text">
        <string>Choose</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
